docker/python/paddle/cloud/dataset/uci_housing.py

```diff
@@ -0,0 +1,30 @@
+import paddle.v2.dataset.uci_housing as uci_housing
+import paddle.v2.dataset.common as common
+import os
+
+__all__=["train", "test", "fetch"]
+
+CURRENT_DATACENTER = os.getenv("CURRENT_DATACENTER")
```

Comment:
Environment variables should be named like "[APPLICATION]_[VARIABLE]" to avoid name conflicts. Like: `PADDLE_CLOUD_CURRENT_DATACENTER`